web/src/components/tasks/business-formation/review/section/ReviewMainSection.tsx

```diff
@@ -1,49 +1,39 @@
 import { UnStyledButton } from "@/components/njwds-extended/UnStyledButton";
 import { LookupStepIndexByName } from "@/components/tasks/business-formation/BusinessFormationStepsConfiguration";
 import { BusinessFormationContext } from "@/contexts/businessFormationContext";
+import { useConfig } from "@/lib/data-hooks/useConfig";
 import { FormationStepNames } from "@/lib/types/types";
 import { scrollToTop } from "@/lib/utils/helpers";
-import { PropsWithChildren, useContext } from "react";
+import { ReactElement, ReactNode, useContext } from "react";
 
 interface Props {
-  buttonText: string;
-  handleButtonClick?: () => void;
   header: string;
   stepName: FormationStepNames;
-  testId: string;
+  testId?: string;
+  children?: ReactNode;
```

Comment:
children shouldn't be optional, right? we would never have a MainSection without anything in it?